src/cript/api/api.py

```diff
@@ -431,14 +431,26 @@ def save(self, new_node):
         quit()
         self._internal_save(new_node, _no_condense_uuid=False)
 
-    def _internal_save(self, new_node: PrimaryBaseNode, _no_condense_uuid: bool) -> None:
-        print("_no_condense_uuid")
-        print(_no_condense_uuid)
+    def _internal_save(self, new_node: PrimaryBaseNode, preknown_uid: str, _no_condense_uuid: bool) -> None:
+        """
+        NOTE: for Ludwig
```

Comment:
@InnocentBug 

Ludwig , this is still Work In Progress, I think I fleshed out most of the skeleton but where to put and use the "uid map" is causing me to scratch my head a bit. I left a pretty detailed note in the docstring here. do you mind taking a a glance at it ? (the note?) and let me know your thoughts?